packages/datadog-plugin-pino/src/index.js

```diff
@@ -112,6 +112,30 @@ module.exports = [
       return this.unwrapExport(pino)
     }
   },
+  {
+    name: 'pino',
+    versions: ['>=6.8.0'],
+    patch (pino, tracer, config) {
+      if (!tracer._logInjection) return
+
+      const mixinSym = pino.symbols.mixinSym
+
+      const wrapped = this.wrapExport(pino, createWrapPino(tracer, config, mixinSym, createWrapMixin)(pino))
```

Comment:
Why not cache `createWrapPino(tracer, config, mixinSym, createWrapMixin)` result ? I think we can reuse the same call for all 3, instead of calling `createWrapPino()` thrice

Reply:
Done.